presto-main/src/main/java/com/facebook/presto/sql/relational/StandardFunctionResolution.java

```diff
@@ -110,6 +113,43 @@ public FunctionHandle arithmeticFunction(ArithmeticBinaryExpression.Operator ope
         return functionManager.resolveOperator(operatorType, fromTypes(leftType, rightType));
     }
 
+    public static boolean isArithmeticFunction(FunctionHandle functionHandle)
```

Comment:
These hurt my eyes. We should probably add `isOperator` and `getOperatorType` to `FunctionMetadata`. Then you can add this to OperatorType as a static function and use like `functionManager.getFunctionMatadata(functionHandle).getOperatorType().map(OperatorType::isArithmeticOperator).orElse(false)`

Reply:
Will add a TODO to these two functions for now.